src/bufr/BufrParser/BufrParser.h

```diff
@@ -51,10 +54,20 @@ namespace Ingester
         /// \brief Exports collected data into a DataContainer
         std::shared_ptr<DataContainer> exportData(const BufrDataMap& sourceData);
 
+        /// \brief Function responsible for splitting the data into the categories. This function
+        ///        gets called over and over for each split speciefied, sub-splitting the data
```

Comment:
Fix typo in specified, and I suggest you add "specified by filter"

Reply:
Thanks for finding the typo. I'm trying to draw a distinction between splits (divide data into sub-parts according to a metric) and filters (throw away data that does not meet a given requirement) so I changed the string to "for each specified Split object".

Reply:
This is a good way of explaining it!

Reply:
Thanks!